### DISCUSSION TITLE
Inside text of enclosing matching pair

### DISCUSSION THREAD

**Discussion by jsammone** [chrisgrieser/nvim-various-textobjs#51] [Q&A] [answered]
Is there a way to get the textobj of the inner text of the enclosing matching pair, the pair itself being generic across [], (), <>, "", '', {}?

Thank you for any help

**Comment by chrisgrieser** [comment]
No, there is not in this plugin, but vim-matchup has such a text object? https://github.com/andymass/vim-matchup#b1-full-set-of-text-objects

**Reply by jsammone** [reply]
This worked pretty close to exactly what I wanted with the quotes enabled option, but it doesn't include <>.

I was able to use mini.ai to get the behavior I wanted.

Thank you for taking the time and pointing me in the right direction!

**Reply by jsammone** [reply]
Oh apologies, you can add in custom characters.

I was able to do `vim.opt.mps:append("<:>")` to include <>.

vim-matchup also has built in highlighting so I ended up going with your suggestion.

Thank you!

**Comment by chrisgrieser** [comment, marked as answer]
By now, the nvim-various-textobjs got two new textobjs, `anyQuote` and `anyBracket` which also address this issue
